libraries/src/Helper/TagsHelper.php

```diff
@@ -825,9 +825,11 @@ public static function getTypes($arrayType = 'objectList', $selectTypes = null,
      *
      * @deprecated  5.3 will be removed in 7.0
      */
-    public function postStoreProcess(TableInterface $table, $newTags = [], $replace = true)
+    public function postStoreProcess(TableInterface $table, $newTags = [], $replace = true): void
```

Comment:
Please remove the :void as well

```suggestion
    public function postStoreProcess(TableInterface $table, $newTags = [], $replace = true)
```

Reply:
Thanks @laoneo

Reply:
The void is still here. Why did this get merged? Its still a bc break.